torch/_dynamo/config.py

```diff
@@ -48,13 +46,13 @@
 
 # Assume these functions return constants
 constant_functions = {
-    torch.jit.is_scripting: False,
-    torch.jit.is_tracing: False,
-    torch._C._get_tracing_state: None,
-    torch.fx._symbolic_trace.is_fx_tracing: False,
-    torch.onnx.is_in_onnx_export: False,
-    external_utils.is_compiling: True,
-    torch._utils.is_compiling: True,
+    "torch.jit.is_scripting": False,
+    "torch.jit.is_tracing": False,
+    "torch._C._get_tracing_state": None,
+    "torch.fx._symbolic_trace.is_fx_tracing": False,
+    "torch.onnx.is_in_onnx_export": False,
+    "torch._dynamo.external_utils.is_compiling": True,
+    "torch._utils.is_compiling": True,
```

Comment:
Cc @yanboliang

Reply:
This is aligned with my refactor, we should use string name rather than function object as much as we can. But I'd suggest to add unit test to check if these hard coding function name is correct by importing module + getattr + eval. Please check this test: https://github.com/pytorch/pytorch/blob/59281d563154b82d60b03702095a3fe3cdd45e98/test/dynamo/test_allow_inline_skip.py#L38

Reply:
Added test for the helper function.